dev-games/godot/godot-3.2.3.ebuild

```diff
@@ -0,0 +1,144 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit python-any-r1 scons-utils desktop xdg
+
+DESCRIPTION="Multi-platform 2D and 3D game engine"
+HOMEPAGE="http://godotengine.org"
+
+SRC_URI="https://github.com/godotengine/${PN}/archive/${PV}-stable.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P}-stable"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="Apache-2.0 BSD BSL-1.1 CC-BY-3.0 MIT MPL-2.0 OFL-1.1 public-domain ZLIB"
+SLOT="0"
+IUSE="+bullet debug +enet +freetype lto +mbedtls +ogg +opus pulseaudio +theora +udev +upnp +vorbis +webp"
+
+RDEPEND="
+	>=app-arch/bzip2-1.0.6-r6
+	>=app-arch/lz4-0_p120
+	>=app-arch/xz-utils-5.0.8
+	>=app-arch/zstd-1.4.4
+	>=dev-libs/json-c-0.11-r1
+	>=media-libs/alsa-lib-1.0.28
+	>=media-libs/flac-1.3.1-r1
+	>=media-libs/libpng-1.6.16:0=
+	>=media-libs/libsndfile-1.0.25-r1
+	>=media-libs/mesa-10.2.8[gles2]
+	>=net-libs/libasyncns-0.8-r3
+	>=sys-apps/attr-2.4.47-r1
+	>=sys-apps/tcp-wrappers-7.6.22-r1
+	>=sys-apps/util-linux-2.25.2-r2
+	>=sys-libs/gdbm-1.11
+	>=sys-libs/glibc-2.20-r2
+	>=sys-libs/libcap-2.22-r2
+	>=sys-libs/zlib-1.2.8-r1
+	>=x11-libs/libX11-1.6.2
+	>=x11-libs/libXcursor-1.1.14
+	>=x11-libs/libXinerama-1.1.3
+	media-libs/alsa-lib
+	dev-libs/libpcre2[pcre32]
+	media-libs/libvpx
+	virtual/glu
+	virtual/opengl
+	bullet? ( >=sci-physics/bullet-2.89 )
+	enet? ( net-libs/enet:= )
+	freetype? ( >=media-libs/freetype-2.5.3-r1:2 )
+	mbedtls? ( net-libs/mbedtls )
+	ogg? ( >=media-libs/libogg-1.3.1 )
+	opus? (
+		media-libs/opus
+		media-libs/opusfile
+	)
+	pulseaudio? ( >=media-sound/pulseaudio-5.0-r7 )
+	theora? ( media-libs/libtheora )
+	udev? ( virtual/udev )
+	upnp? ( net-libs/miniupnpc )
+	vorbis? ( >=media-libs/libvorbis-1.3.4 )
+	webp? ( media-libs/libwebp )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_prepare() {
+	default
+	rm -rf thirdparty/{bullet,enet,freetype,libogg,libpng,libtheora,libvorbis,libvpx,libwebp,mbedtls,miniupnpc,opus,pcre2,zlib,zstd} || die
```

Comment:
Remove -f, otherwise it'll succeed even if didn't remove anything, so ||die would not help